src/redux/Auth/auth-reducer.js

```diff
@@ -0,0 +1,49 @@
+import * as actionTypes from "../shopping-types";
+
+export const authReducer = (state = { data: null }, action) => {
```

Comment:
```suggestion
export const authReducer = (state = { data: null, loading: false, error: null }, action) => {
```

Apply this to all reducers